src/abstract-interpretation/data-frame/absint-visitor.ts

```diff
@@ -142,11 +146,10 @@ class DataFrameAbsintVisitor<
 					getOriginInDfg(this.config.dfg, operation.operand)
 						?.filter(origin => origin.type === OriginType.ReadVariableOrigin)
 						.forEach(origin => this.newDomain.set(origin.id, value));
+				} else if(getConstraintType(operation.operation) === ConstraintType.ResultPostcondition) {
+					this.newDomain.set(node.info.id, value);
```

Comment:
Overwriting on a post condition is fine, but we should discuss cases in which the assignment is, e.g., delayed